django_spanner/schema.py

```diff
@@ -117,14 +117,22 @@ def create_model(self, model):
             # Create a unique constraint separately because Spanner doesn't
             # allow them inline on a column.
             if field.unique and not field.primary_key:
-                self.deferred_sql.append(
-                    self._create_unique_sql(model, [field.column])
-                )
+                if USING_DJANGO_3:
```

Comment:
with django 3, originally it calls `self._create_unique_sql(model, [field.column])` and now it calls `self._create_unique_sql(model, [field])` is it actually right?